ci-operator/step-registry/ipi/conf/gcp/xpn/ipi-conf-gcp-xpn-commands.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+PATCH=/tmp/install-config-xpn.yaml.patch
+
+HOST_PROJECT="openshift-dev-installer"
+HOST_PROJECT_CONTROL_SUBNET="installer-shared-vpc-subnet-1"
+HOST_PROJECT_COMPUTE_SUBNET="installer-shared-vpc-subnet-2"
+HOST_PROJECT_NETWORK="installer-shared-vpc"
+
+cat > "${PATCH}" << EOF
+featureSet: TechPreviewNoUpgrade
```

Comment:
Let's get openshift/installer#6754 merged and then remove tech preview. 

Hopefully that will make the rehearsals more stable.